modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -251,7 +271,6 @@ public void processOpts() {
         }
 
         if (!withoutRuntimeChecks) {
-            this.modelTemplateFiles.put("models.mustache", ".ts");
```

Comment:
@TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01) @topce (2018/10) @akehir (2019/07) @petejohansonxo (2019/11) @amakhrov (2020/02) @davidgamero (2022/03) @mkusaka (2022/04)

fyi. with this change, the users cannot skip model files with the `withoutRuntimeChecks` option but they can still do so using openapi generator ignore file.

Reply:
Yep, added this back